README.md

```diff
@@ -10,19 +10,38 @@ Inspired by a pattern seen in [Polar](http://www.polarb.com/), [IE 10+](http://i
 
 The plugin works in any browser that supports resetting the `type` attribute of `<input>` elements (pretty much everything newer than IE8). The plugin should fall back gracefully in cases where this is not supported.
 
-If [Bower](http://bower.io/)'s your thing, you can install this plugin by running `bower install hideShowPassword` in your project directory.
+## Installation
 
-## Dependencies
+Simply include the plugin after you've included [jQuery](http://jquery.com/):
 
-hideShowPassword requires [jQuery](http://jquery.com/).
+```html
+<script src="//code.jquery.com/jquery-2.1.3.min.js"></script>
+<script src="path/to/hideShowPassword.min.js"></script>
+```
+
+If [Modernizr](http://modernizr.com/) is also included, the plugin's touch enhancements will default to the value of `Modernizr.touch`.
 
-### Optional: Modernizr
+### Using [npm](https://www.npmjs.com/) and [Browserify](http://browserify.org/)
+
+```
+npm install --save jquery
+npm install --save hideShowPassword
+```
 
-If [Modernizr](http://modernizr.com/) is included, the plugin's touch enhancements will default to the value of `Modernizr.touch`.
+```javascript
+var $ = require('jquery');
+require('hideShowPassword');
+```
```

Comment:
I tried both, and both worked fine. The plugin does its own `require` call, and npm appears to maintain state of a module across `require` calls as long as the modules are not in opposing `package.json` objects. (So if I were to include `jquery` in `devDependencies` and `hideShowPassword` in `dependencies`), `require('jquery')` would return a _separate_ jQuery instance from the one included from the plugin... crazy stuff!)